backend/sites/src/app/services/dropdown/dropdown.service.ts

```diff
@@ -60,80 +75,101 @@ export class DropdownService {
         });
       }
 
-      // Order the results by 'id' in ascending order
       queryBuilder.orderBy('people_orgs.displayName', 'ASC');
 
-      // Execute the query and get the results
       const result = await queryBuilder.getMany();
 
-      // Return formatted results
-      if (result) {
-        return result.map((obj: any) => ({
+      return (
+        result.map((obj: PeopleOrgs) => ({
           key: obj.id,
           value: obj.displayName,
-        }));
-      } else {
-        return []; // Return an empty array if no results
-      }
+        })) || []
+      );
     } catch (error) {
-      throw new Error('Failed to retrieve people orgs.');
+      throw new Error('Failed to retrieve people organizations.');
     }
   }
 
+  /**
+   * Retrieves notation type codes and organizes them by metadata.
+   *
+   * @returns A promise that resolves to an array of objects with `metaData` and `dropdownDto` properties.
+   * @throws Error if there is an issue retrieving the data.
+   */
   async getNotationTypeCd() {
     try {
       const result = await this.eventTypeCdRepository.find();
       if (result) {
```

Comment:
all if should have a corresponding else block

Reply:
outside of block there is a return [] if there will be no result in any case it will return empty array, which is handled in resolver